lib/auth/permissions.go

```diff
@@ -171,18 +171,20 @@ func GetCheckerForBuiltinRole(role teleport.Role) (services.AccessChecker, error
 			services.RoleSpecV2{
 				Namespaces: []string{services.Wildcard},
 				Resources: map[string][]string{
-					services.KindProxy:         services.RW(),
-					services.KindOIDCRequest:   services.RW(),
-					services.KindOIDC:          services.RO(),
-					services.KindNamespace:     services.RO(),
-					services.KindEvent:         services.RW(),
-					services.KindSession:       services.RW(),
-					services.KindNode:          services.RO(),
-					services.KindAuthServer:    services.RO(),
-					services.KindReverseTunnel: services.RO(),
-					services.KindCertAuthority: services.RO(),
-					services.KindUser:          services.RO(),
-					services.KindRole:          services.RO(),
+					services.KindProxy:                 services.RW(),
+					services.KindOIDCRequest:           services.RW(),
+					services.KindOIDC:                  services.RO(),
+					services.KindNamespace:             services.RO(),
+					services.KindEvent:                 services.RW(),
+					services.KindSession:               services.RW(),
+					services.KindNode:                  services.RO(),
+					services.KindAuthServer:            services.RO(),
+					services.KindReverseTunnel:         services.RO(),
+					services.KindCertAuthority:         services.RO(),
+					services.KindUser:                  services.RO(),
+					services.KindRole:                  services.RO(),
+					services.KindClusterAuthPreference: services.RO(),
+					services.KindUniversalSecondFactor: services.RO(),
```

Comment:
Why does "2nd factor" deserves to be a top-level thing? Isn't it one of many sub-properties of ClusterAuthPreference?